README.md

```diff
@@ -1,5 +1,14 @@
 # @nativescript/\* plugins
 
+## Prerequisites
+
+```
+npm i -g nps nps-utils nps-i
+export NODE_PATH=`npm root -g`
+```
+
+## Run
+
 ```
 npm run setup
```

Comment:
@crowmagnumb `npm run setup` installs all workspace dep's like `nps-utils` and nothing needs to be installed globally. Did you do `npm run setup` first? If so could you provide the output of just running that on your end?

Reply:
Well damnit. Of course I did that. :) Then when running `npm run start` I got the errors about the missing nps, nps-utils, nps-i. I even checked the `node_modules/.bin` directory and did not find them there like I thought I might. That's when I went ahead and installed them globally. So now, to try and recreate this I ran `npm un -g nps nps-utils nps-i` and then re-ran `npm run setup`. And this time it worked fine and the executables are in the `node_modules/.bin` directory.

Sheesh, sorry I don't know what to say. :(